.github/workflows/cd-publish.yml

```diff
@@ -3,38 +3,98 @@ name: CD - Publish for Blazor / FluentUI
 on:
   workflow_dispatch
 
+env:
+  OUTPUT_PATH: ${{ github.workspace }}/.nupkg
+  NUGET_SOURCE: "https://api.nuget.org/v3/index.json"
+  DOTNET_VERSION: "5.0.5"
+  GITHUB_SOURCE: "https://nuget.pkg.github.com/Azure/index.json"
+  CONFIGURATION: Release
+
 jobs:
-  publish:
+  build job:
     name: setup, build, validate, and publish
     runs-on: windows-latest
     if: contains('nicholasrice,awentzel,eisenbergeffect,Ibrahimmaga', github.actor)
 
     steps:
     - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-        
-    - name: Setup .NET
+    - name: Setup .NET Core ${{ env.DOTNET_VERSION }}
```

Comment:
eliminate specific versions.  we don't need this.

Reply:
okay